docs/source/portfolio_optimisation/theory_implied_correlation.rst

```diff
@@ -0,0 +1,164 @@
+.. _portfolio_optimisation-theory_implied_correlation:
+
+
+================================
+Theory-Implied Correlation (TIC)
+================================
+This TIC class includes an algorithm to calculate the Theory-Implied Correlation and a method to calculate the correlation matrix distance proposed by Herdin and Bonek. This distance may be used to measure to which extent the TIC matrix has blended theory-implied views (tree structure of the elements) with empirical evidence (correlation matrix).
```

Comment:
Please add link to this paper: Herdin and Bonek

Reply:
Added links to the papers in the intro